### DISCUSSION TITLE
Do not implement paging until after the initial release

### DISCUSSION THREAD

**Discussion by youyingfeng** [youyingfeng/Eidos#37] [Ideas]
The size of a work is likely to be very small, so loading many of these should not cause memory problems.

How small is small?

One page of AO3 work blurbs (20 blurbs) takes up about 200kb of space (HTML only). This is including all the HTML comments and cruft in the file. Then, a single work blurb would take up around 20kb of space - 40kb for a very very conservative estimate - in memory. Loading 2000 of these work blurbs would take up 80mb of space - a significant amount, but not shockingly bad (according to my limited knowledge anyway). While this scenario is possible, it is unlikely to happen, and this issue is thus not a priority. It can probably be resolved after the initial release has been made, when bug reports start surfacing.

Furthermore, implementing paging is a painful process - this is not an easy library. Integration with SQLDelight is also in beta, so it would be wise to hold off on it for now.

In short: Do not implement paging (specifically using the paging library) until after the initial release.
